app/services/course/assessment/question/programming/programming_package_service.rb

```diff
@@ -45,8 +45,9 @@ def init_language_package_service(params)
         Course::Assessment::Question::Programming::Python::PythonPackageService.new params
     elsif javascript_language
       raise NotImplementedError
-    else
-      raise NotImplementedError
+    elsif cpp_language
```

Comment:
At this rate, can we switch to a case statement instead, and remove all the methods? So it should look something like the one below. Also, you need to add back the case to catch all other languages. 

```ruby
def init_language_package_service(params)
  @language_package_service = begin
    case @language.class
    when Coursemology::Polyglot::Language::Python
      Course::Assessment::Question::Programming::Python::PythonPackageService.new params
    when ...
      ...
  end
end
```